web/yo/app/scripts/controllers/gene.js

```diff
@@ -3069,6 +3069,26 @@ angular.module('oncokbApp')
                 })
                 .finally(getSuggestedMutations);
 
+            storage.retrieveMeta().then(function(result){
+                storage.getMetaRealtimeDocument(result[0].id).then(function (metaRealtime) {
+                    if(metaRealtime && metaRealtime.error) {
+                        console.log('did not get meta document.');
```

Comment:
If we did not get meta document, we should alert the curator to contact us and disable all other editing features.